src/components/tableRow.jsx

```diff
@@ -112,7 +126,11 @@ class TableRow extends Component<Object> {
     }
 
     return (
-      <Table.Row negative={isError} className={basic === "very" ? classes.veryBasicGrid : ""}>
+      <Table.Row
+        negative={isError}
+        onClick={() => this.handleRowClick()}
```

Comment:
Please change this line

onClick={this.handleRowClick}

https://reactjs.org/docs/faq-functions.html#bind-in-constructor-es2015